great-idea/css/index.css

```diff
@@ -93,7 +93,13 @@ h1, h2, h3, h4, h5 {
 	color: grey;
 	text-decoration: none;
 }
-.top .logo {
+.top a .logo {
+	/* background-color: blue;	 */
+	display: flex;
+	margin-right: 0%; 
```

Comment:
If the value is 0 or 0%, there's no need to add anything at all. We usually want to write as little code as possible, as added code increases the risk of bugs.